src/main/java/io/zeebe/http/variables/EnvironmentVariablesProvider.java

```diff
@@ -0,0 +1,46 @@
+package io.zeebe.http.variables;
+
+import io.zeebe.http.ZeebeHttpWorkerConfig;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public abstract class EnvironmentVariablesProvider {
+  protected final ZeebeHttpWorkerConfig config;
+
+  protected EnvironmentVariablesProvider(ZeebeHttpWorkerConfig config) {
+    this.config = config;
+  }
+
+  abstract protected Map<String, String> getRawVariables();
+
+  /**
+   * @return Only return environment variables starting with the configured
+   * prefix and (optionally) removing it.
+   */
+  public Map<String, String> getVariables() {
+    return getRawVariables().entrySet().stream()
+      .filter(this::startsWithPrefix)
```

Comment:
I think that this filter would break the current behavior. By default, the prefix is set to `ZEEBE_ENV_`. If it is not set (e.g. in Camunda cloud) then the variables are not visible anymore.

Reply:
I assume that the filter and remove prefix is only useful for the local environment variables. So, I would move it to the `LocalVariableProvider` instead of having this abstract class.

Reply:
ok, I'll change that

Reply:
done